tools/appliance/definitions/systemvmtemplate/configure_systemvm_services.sh

```diff
@@ -39,12 +39,76 @@ function install_cloud_scripts() {
     /etc/init.d/{cloud,cloud-early-config,cloud-passwd-srvr,postinit} \
     /etc/profile.d/cloud.sh
 
-  chkconfig --add cloud-early-config
-  chkconfig cloud-early-config on
-  chkconfig --add cloud-passwd-srvr
-  chkconfig cloud-passwd-srvr off
-  chkconfig --add cloud
-  chkconfig cloud off
+  cat > /lib/systemd/system/cloud-early-config.service << EOF
```

Comment:
We can do that, but isn't the files in /etc/systemd/system/multi-user.target.wants symlinked to /lib/systemd? I'm open to changes as long as the stuff works.